src/sensai/evaluation/__init__.py

```diff
@@ -3,4 +3,4 @@
 from .eval_util import RegressionEvaluationUtil, ClassificationEvaluationUtil, MultiDataEvaluationUtil, \
     evalModelViaEvaluator, createEvaluationUtil, createVectorModelEvaluator, createVectorModelCrossValidator
 from .evaluator import VectorClassificationModelEvaluator, VectorRegressionModelEvaluator, \
-    VectorRegressionModelEvaluationData, VectorClassificationModelEvaluationData
```

Comment:
Why have so many classes in the evaluation modules been renamed?

Reply:
As a preparation for tensor model - I will use this classes in the TensorModelEvaluator. If you want, I can revert the change for the time being. The EvaluationData classes seemed to me more general than VectorModel, therefore the renaming

Reply:
If you want, I can revert this for the time being

Reply:
In preparation for tensor model - I will base the evaluation classes for tensor model from ModelEvaluationData

> Please use more detailed commit messages (not just in this PR, but in general). For example, "Fixed bug in XY" or "Improvements in XY" does not contain a lot of information. What was the bug, how was it fixed, what is the improvement?

Right, I got sloppy. I will reword them a bit

Reply:
> If you want, I can revert this for the time being

If you can easily revert, I would prefer this until we are ready to reassess the needs as new types of models are implemented.